resources/js/components/ui/form/SegmentedControl.tsx

```diff
@@ -0,0 +1,79 @@
+import * as radio from "@zag-js/radio-group";
+import {normalizeProps, useMachine} from "@zag-js/solid";
+import {cx, htmlAttributes} from "components/utils";
+import {For, VoidComponent, createComputed, createMemo, createUniqueId, splitProps} from "solid-js";
+import {FieldLabel} from ".";
+import s from "./SegmentedControl.module.scss";
+
+interface Props extends htmlAttributes.div {
+  name: string;
+  items: Item[];
+  label?: string;
+  /** Optionally value, if should be used in standalone mode (not in form). */
+  value?: string;
+  setValue?: (value: string) => void;
+  disabled?: boolean;
+  /** Whether to make the control smaller. */
+  small?: boolean;
+}
+
+interface Item {
+  readonly value: string;
+  readonly text: string;
+}
+
+/**
+ * A segmented control component, which is suitable for use both standalone and in a form.
+ *
+ * To use in a form, do not specify the value and setValue props. The component will connect to
+ * the form using the native input elements.
+ *
+ * To use in standalone mode, specify the controlling signal via value and setValue.
+ */
+export const SegmentedControl: VoidComponent<Props> = (props) => {
```

Comment:
Co to segmented control i gdzie to będzie? Daj jakiś screenshot.

Reply:
Dodałem demo na stronie kalendarza admina placówki (tam gdzie tiny calendar). 
![image](https://github.com/mblajek/Memo/assets/115690/1baef07a-1a1d-4213-bc1f-efc0f5ec32d1)

Reply:
Ja się na wyglądzie nie znam, ale jakiś piękny to się nie wydaje. :p